config/serverless.yml

```diff
@@ -1,2 +1,2 @@
 xpack.fleet.enableExperimental: ['fleetServerStandalone']
-xpack.fleet.internal.ILMPoliciesDisabled: true
```

Comment:
Can we find out why this causes problems on startup instead of deleting it?

Reply:
For some reason this disappeared for a while but then caused an error again in my recent test :thinking:

Reply:
Got it, it seems there was a mismatch between the set variable and the config validation when it has been introduced[ here](https://github.com/elastic/kibana/pull/154460/files#diff-ebeb9e9dc4c3ee5e81f743f0eda3c06f1fc161a11127b374d21f0da5e6368fd9). Updated to match the validation.

Reply:
nice find, thank you! I guess nobody else tried to start kibana with `--serverless` so far? :raised_eyebrow:

Reply:
That change is 9 days old, possible? 😅